.github/workflows/CI.yml

```
@@ -195,12 +195,16 @@ jobs:
   # we might want to pin the SHA once merged
   coverage-comment:
     # Important: make sure to update the SHA after making any changes to the coverage workflow
-    uses: pydata/pydata-sphinx-theme/.github/workflows/coverage.yml@4a1e7898d6c92dade5e489684277ab4ffd0eb053
+    # removing pin to test changes
+    uses: ./.github/workflows/coverage.yml
     needs: [coverage]
     # ensures this runs even if the coverage step does not continue - e.g. the
     # default coverage action will fail at first for external PRs, this is a workaround
     # to ensure the comment is posted
-    if: ${{ always() }}
+    if: ${{ github.event.name == 'pull_request'}}
+    with:
+      # this is the workflow run ID of the CI workflow
```

```suggestion
      # this is the workflow run ID of the `CI.yml` workflow
```
(for clarity. On first read I thought "ok, but _which_ CI workflow?")